data/comparisons/prometheus-vs-grafana.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Prometheus vs Grafana - Detailed Comparison for 2024
+title: Prometheus vs Grafana - Which Tool is Right for You?
 slug: prometheus-vs-grafana
 date: 2024-02-27
```

Comment:
```suggestion
date: 2024-06-11
```

@yuvraajsj18 we should also update the date when we update an old blog.

Reply:
Noted.